source/app/service-providers/commands/language-tool.ts

```diff
@@ -118,7 +119,7 @@ export default class LanguageTool extends ZettlrCommand {
    *
    * @return  {Promise<LanguageToolAPIResponse|string|undefined>}       The result
    */
-  async run (evt: string, arg: { text: string, language: string }): Promise<[LanguageToolAPIResponse, string[]]|string|undefined> {
```

Comment:
If I saw it correctly, we won't need `text` here anymore, correct? It's only used for the character count, but that's easy to calculate from the `data` property.

Also: Please don't stringify that in the renderer, and rather do that here and properly type that property.